types.go

```diff
@@ -224,3 +236,138 @@ type AccountAssociatedAsset struct {
 	Unit     string `json:"unit,omitempty"`
 	Quantity string `json:"quantity,omitempty"`
 }
+
+// MetadataTxLabel return Transaction metadata labels
+// List of all used transaction metadata labels.
+type MetadataTxLabel struct {
+	Label string `json:"label,omitempty"`
+	Cip10 string `json:"cip10,omitempty"`
+	Count string `json:"count,omitempty"`
+}
+
+// MetadataTxContentInJSONRaw Transaction metadata content raw in JSON
+// Transaction metadata per label.
+// This struct are more flexible on JSONMetadata field
+type MetadataTxContentInJSONRaw struct {
+	TxHash string `json:"tx_hash,omitempty"`
+	// 	string or object or Array of any or integer or number or boolean Nullable
+	// Content of the JSON metadata
+	JSONMetadata map[string]interface{} `json:"json_metadata,omitempty"`
+}
+
+// MetadataTxContentInJSON return Transaction metadata content in JSON
+// Transaction metadata per label.
+// This struct are stronger typed on JSONMetadata field
+type MetadataTxContentInJSON struct {
+	TxHash       string       `json:"tx_hash,omitempty"`
+	JSONMetadata JSONMetadata `json:"json_metadata,omitempty"`
+}
+
+type JSONMetadata struct {
+	Adausd []Adausd `json:"ADAUSD,omitempty"`
+}
+
+type Adausd struct {
+	Value  string `json:"value,omitempty"`
+	Source string `json:"source,omitempty"`
+}
+
+// MetadataTxContentInCBOR return Transaction metadata content in CBOR
+// Transaction metadata per label.
+type MetadataTxContentInCBOR struct {
+	TxHash       string `json:"tx_hash,omitempty"`
+	CborMetadata string `json:"cbor_metadata,omitempty"`
+}
+
+// List of stake pools
+// List of registered stake pools.
+type Pools []string
+
+// List of retired stake pools
+// List of already retired pools.
+type PoolRetired struct {
+	PoolID string `json:"pool_id,omitempty"`
+	Epoch  int    `json:"epoch,omitempty"`
+}
+
+type PoolRetiring struct {
```

Comment:
Do we need PoolRetiring and PoolRetired? They both look the same.

Reply:
Both have the same struct data type. In the future the API will be change? What is your appreciation ?

Reply:
Just had a look at the schema. We could keep this as is since the schema has them separate as well.